data/defaults.yaml

```diff
@@ -110,7 +110,7 @@ prometheus::haproxy_exporter::group: 'haproxy-exporter'
 prometheus::haproxy_exporter::package_ensure: 'latest'
 prometheus::haproxy_exporter::package_name: 'haproxy_exporter'
 prometheus::haproxy_exporter::user: 'haproxy-user'
-prometheus::haproxy_exporter::version: '0.7.1'
+prometheus::haproxy_exporter::version: '0.9.0'
```

Comment:
can you please add an acceptance test that first explictly installs 0.7.1 and afterwards upgrades it to 0.9.0? We already have a few tests in this repo that do the same for other exporters.

Reply:
Makes sense. I'll check this asap.